src/platform/forms-system/src/js/components/FieldTemplate.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import get from '../../../../utilities/data/get';
 import classNames from 'classnames';
+import get from '../../../../utilities/data/get';
```

Comment:
@brdunfield did you mean to make a change to this file?

Reply:
I did not. Reverted!